### ISSUE TITLE
Outline is cut for Dropdown widgets with non-default border radius

### ISSUE THREAD

**Issue by last-partizan** [formkit/formkit#1099]
### Reproduction

https://themes.formkit.com/editor?variables=radius=rounded-full,accentColor=blue,colorTemperature=neutral,spacing=2,scale=base,inputShadow=shadow,baseColorShade=600,borderShadeLightMode=400,borderShadeDarkMode=500,inputMaxWidth=max-w-[20em],tagRadius=rounded,swatchRadius=rounded&theme=regenesis

### Describe the bug

This is most obvious on the Autocomplete widgets with single selection:

![image](https://github.com/formkit/formkit/assets/301015/af658248-7709-47a5-bc13-67ebbd203ec8)

But also reproducible for any selected element if it's first or last in the list.

### Environment

• OS: Linux
• Browser: Firefox
• Version: 1.4.0

**Comment by last-partizan** [comment]
I think, this can be fixed on the theme level.

But, until it's fixed i wanted to do a local fix by overriding `classes`.

I have tried this, but it doesn't work. How to properly pass path to dropdown listitem element? I have also tried using `generateClasses`, it provides a way to specify which widget i want to style, but it does not support object syntax, and i like to use object syntax, because it's much better than plain text.

```ts
export const formKitConfig = defaultConfig({
  config: {
    rootClasses,
    classes: {
      'family:dropdown__listitem': {
        $reset: true,
        'text-red': true,
      },
    },
  },
})
```
